### DISCUSSION TITLE
Over dosing chlorine (ORP)

### DISCUSSION THREAD

**Discussion by flyboyusa** [tagyoureit/nodejs-poolController#1009] [Nixie]
Maybe I'm missing something....but my Nixie seems to continue to dose chlorine via my dosing pump, even when the ORP reading is over the target.
What's the rationale for when Nixie stops dosing?
<img width="812" alt="Screenshot 2024-07-30 at 9 59 40 PM" src="https://github.com/user-attachments/assets/8eaa2b8e-78dc-4a89-a05e-55ee644c1862">
<img width="378" alt="Screenshot 2024-07-30 at 10 00 04 PM" src="https://github.com/user-attachments/assets/9f8da027-d3a1-41f3-a053-555d791af236">

**Comment by tagyoureit** [comment]
Can you attach a replay?  And/or share some screenshots of your chemcontroller setup?
